ui/helpers/constants.js

```diff
@@ -217,6 +217,11 @@ export const thPlatformMap = {
     'Android 8.0 Pixel2 AArch64 WebRender',
   'android-hw-p2-8-0-android-aarch64-shippable-qr':
     'Android 8.0 Pixel2 AArch64 Shippable WebRender',
+  'android-hw-a51-11-0-arm7-qr': 'Android 11.0 Samsung A51',
+  'android-hw-a51-11-0-arm7-shippable-qr': 'Android 11.0 Samsung A51 Shippable',
+  'android-hw-a51-11-0-aarch64-qr': 'Android 11.0 Samsung A51 AArch64',
```

Comment:
We left `Google` out and only used `Pixel` but in this case keeping the brand sounds useful to prevent confusion with chips and architectures.